manifests/config.pp

```diff
@@ -80,6 +80,34 @@
     ensure  => absent,
   }
 
+  if $::foreman::jobs_sidekiq {
+    if $::foreman::jobs_sidekiq_redis_url == undef {
+      include ::redis
+
+      redis::instance { 'dynflow':
```

Comment:
Does an instance refer to a "database" inside Redis or too the Redis process itself? If the latter, will this tie all future deployments and use of Redis (e.g. caching or Pulp 3) to an instance named for dynflow?

Reply:
The `instance` is completly separate proces, with its own config and process. So if we decide to use redis for caching, there should be another instance. This should mean dynflow has it's own instance and configuration - meaning not sharing it with other use cases of redis.

Reply:
I suppose we can clean that up in the future as I think having multiple Redis instances may not be the cleanest design or requirement for our users. But generally I think I would prefer a single instance for Foreman if possible.